File: EjemploCodigo/Java/SubClasesCar/UberX.java
package Java.SubClasesCar;

import Java.Account;
import Java.Car;

public class UberX extends Car{
    
    String brand;
    String model;

    public UberX(String license, Account driver, String brand,  String model){
        super(license, driver);//llamo al metodo consturctor de Car ->  esto es por que Car tiene un constructor 
        this.brand = brand;
        this.model  = model;
    }
}

File: EjemploCodigo/PHP/SubClasesCar/UberX.php
<?php
require_once('../Car.php');

class UberX extends Car{
    public $brand;
    public $model;
    

    public function __construct($license, $driver, $brand, $model ) {
        parent ::__construct($license, $driver); //llamo al metodo consturctor de Car ->  esto es por que Car tiene un constructor 
        $this->brand = $brand;
        $this->model = $model;
    }

}
